tests/GiraffeGenerator.IntegrationTests/RemoveGenerated.ps1

```diff
@@ -0,0 +1 @@
+﻿ls Spec*.fs | ?{ -not $_.Name.EndsWith("Tests.fs") } | rm
```

Comment:
Why do we need this script?

Reply:
I use it to remove all generated files without changing the project to re-generate them. Comes handy on massive changes.